pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -1086,9 +1039,9 @@ func populateInitialStatusFromVerified(ctx *verifierContext,
 }
 
 func getVerifierDir(objType string) string {
-	return verifierBasePath + "/" + objType + "/" + "verifier"
+	return path.Join(verifierBasePath, "verifier", objType)
 }
 
 func getVerifiedDir(objType string) string {
-	return verifierBasePath + "/" + objType + "/" + "verified"
+	return path.Join(verifierBasePath, "verified", objType)
```

Comment:
Why swap the order?

Reply:
No good reason. This is an artifact of when I tried to remove `objType` entirely, and had to defer, so I put it back. I will fix that ordering.

Reply:
Fixed